plugins/modules/openvswitch_bridge.py

```diff
@@ -147,7 +147,7 @@ def map_obj_to_commands(want, have, module):
                             command += " " + k + " " + v
                             commands.append(command)
 
-            if want["vlan"] and want["vlan"] != have["vlan"]:
+            if want["vlan"] and str(want["vlan"]) != have["vlan"]:
```

Comment:
Please use the to_text() method instead of str() here.
`from ansible.module_utils._text import to_text`